spec/lib/vcr/errors_spec.rb

```diff
@@ -22,7 +22,7 @@ def request_with(values)
         )
       end
 
-      context 'when there is no current cassette' do
+      context 'when there is no cassette' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.